.swiftlint.yml

```diff
@@ -0,0 +1,15 @@
+# Realm Swift's SwiftLint configuration file
+# Uncomment the commented lines to avoid any violations
+
+included:
+  - RealmSwift-swift2.0
+disabled_rules:
+  - force_cast
+  - force_try
+  - todo
+line_length:
+  - 120
+file_length:
+  - 670
+type_body_length:
+  - 581
```

Comment:
They came from the existing file & type body lengths, so that the current sizes would pass, but exceeding that in the future would not. That reasoning was stupid, so I've now disabled those rules entirely.